transformers/component_tests/jug/transformer_test.go

```diff
@@ -36,18 +39,18 @@ import (
 var _ = Describe("Executing the transformer", func() {
 	var (
 		db                = test_config.NewTestDB(test_config.NewTestNode())
-		err               error
-		ilkID             int64
-		storageKeysLookup = storage.NewKeysLookup(jug.NewKeysLoader(&mcdStorage.MakerStorageRepository{}))
-		repository        = jug.JugStorageRepository{}
-		contractAddress   = "25a008bf942ce6d5b362f91ed7ae3e4104286a12"
+		contractAddress   = test_data.JugAddress()
+		storageKeysLookup = storage.NewKeysLookup(jug.NewKeysLoader(&mcdStorage.MakerStorageRepository{}, contractAddress))
+		repository        = jug.JugStorageRepository{ContractAddress: contractAddress}
 		transformer       = storage.Transformer{
 			HashedAddress:     vdbStorage.HexToKeccak256Hash(contractAddress),
 			StorageKeysLookup: storageKeysLookup,
 			Repository:        &repository,
 		}
-		headerID int64
 		header   = fakes.FakeHeader
+		headerID int64
```

Comment:
Super minor and not introduced by this PR but I've become a fan of not having a `headerID` var and instead always accessing that val via `header.Id` in tests so that we don't get weird failures deriving from `headerID != header.Id`